src/resources/BreadCrumbResource.ts

```diff
@@ -66,3 +66,11 @@ export function getStaffDashboardBreadcrumb (): BreadcrumbIF {
     href: `${sessionStorage.getItem('BUSINESSES_URL')}staff/${getParams()}`
   }
 }
+
+export function getStaffRegistryBreadcrumb (): BreadcrumbIF {
```

Comment:
It's actually not the "Registry" page (which is https://www.bcregistry.gov.bc.ca/) but the "(My) Business Registry" page

Reply:
Obsolete. Resolving.